### ISSUE TITLE
LPD8 noteOff sends vel 127, which breaks noteOnOff shortcut

### ISSUE THREAD

**Issue by adcxyz** [ModalityTeam/Modality-toolkit#251]
LPD8 sends noteOff with vel 127, which makes it impossible to detect noteOff by el.value == 0;
* is this only my (adc) device, or do others get the same behavior?
* if this is general, how best to handle this case? separate noteOn, noteOff elements?
```
m = MKtl(\l, "*lpd8");
MIDIFunc.trace; // hit pad: noteOn has proper velocity, noteOff is always 127 - why?
m.gui; m.trace;
m.elAt(\pad, 0).action = { |el| [el.name, el.value].postln; };
m.elAt(\pad, 0).value; // always 1.0 after releasing pad
m.device.midiKeyToElemDict[\non_0_38] === m.device.midiKeyToElemDict[\nof_0_38]
```

**Comment by adcxyz** [comment]
fixed with 178d6de

**Comment by LFSaw** [comment]
IMHO, this should be the default way to implement noteOn / noteOff for pad/keyboard devices. I now have the same problem with the OP-1 (sends non-zero noteOff events which I cannot differentiate from noteOn's)
